src/main/java/uk/ac/sanger/sccp/stan/request/WorkProgress.java

```diff
@@ -101,16 +112,17 @@ public boolean equals(Object o) {
         if (o == null || getClass() != o.getClass()) return false;
         WorkProgress that = (WorkProgress) o;
         return (Objects.equals(this.work, that.work)
-                && Objects.equals(this.timestamps, that.timestamps));
+                && Objects.equals(this.timestamps, that.timestamps))
+                && this.mostRecentOperation==that.mostRecentOperation;
```

Comment:
Objects.equals(this.mostRecentOperation, that.mostRecentOperation)

Reply:
Ive seen this syntax in other equals methods, just wondering the difference / benefits for Objects.equals over this